packages/paste-core/components/chat-composer/src/ChatComposer.tsx

```diff
@@ -29,37 +28,22 @@ import {
   HistoryPlugin,
 } from '@twilio-paste/lexical-library';
 import {StylingGlobals} from '@twilio-paste/styling-library';
-import type {LexicalComposerProps, OnChangeFunction, ContentEditableProps} from '@twilio-paste/lexical-library';
+import type {LexicalComposerProps, ContentEditableProps, OnChangeFunction} from '@twilio-paste/lexical-library';
 import merge from 'deepmerge';
 
 import {chatComposerLexicalStyles} from './styles';
 import {AutoLinkPlugin} from './AutoLinkPlugin';
 import {PlaceholderWrapper} from './PlaceholderWrapper';
 import {baseConfig, renderInitialText} from './helpers';
 
-export interface ChatComposerProps extends ContentEditableProps {
+export interface ChatComposerProps extends Omit<ContentEditableProps, 'style' | 'className' | 'onChange'> {
   children?: LexicalComposerProps['children'];
   config: LexicalComposerProps['initialConfig'];
   element?: BoxProps['element'];
-  placeholder?: string | JSX.Element;
```

Comment:
unsure on this one, it clashes now and I don't really know why we would change it from what is supplied in the ContentEditable types